### PR TITLE
space ship done

### LINKED ISSUE #15: Space Ship Minigame
# REQUIREMENTS
the minigame will use a channel
just like horses, players will make an initial bet
and the game will start
players can get out of the ship whenever they want and get paid (the reward will be higher the more time passes), or risk the ship exploding and lose the bet

### LINKED ISSUE #16: Place user coins in navbar
# REQUIREMENTS
Place user coins in navbar and write the javascript to update the coins in real time

### PR DESCRIPTION
**Pull request by MrCano369x**
Co-authored-by: alexs <alexs1010@gmail.com>

# Description
What did you implemented/Why did you implemented this:
we made the spaceship game, and put the user's coins in the navbar

  ## Screenshots 
image

### CODE REVIEW

File: app/assets/stylesheets/space_ship.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.space {
```
Comment (Frcomparan) [inline app/assets/stylesheets/space_ship.css:1]: Please, sort alphabetically your styles in this file

File: app/assets/stylesheets/space_ship.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  margin-top: 20px;
+}
+
+.bets{
+  border-collapse: collapse;
+  height: min-content;
+  background-color: black;
```
Comment (Frcomparan) [inline app/assets/stylesheets/space_ship.css]: Here

File: app/assets/stylesheets/space_ship.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  display: none;
+}
+
+.show{
+  display: inline;
+}
+
+#btnLeft{
+  height: min-content;
+  align-self: center;
```
Comment (Frcomparan) [inline app/assets/stylesheets/space_ship.css]: Here

File: app/assets/stylesheets/space_ship.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+#ship{
+  position: relative;
+  animation: move 2s ease-in-out infinite alternate;
```
Comment (Frcomparan) [inline app/assets/stylesheets/space_ship.css]: Here

File: app/assets/stylesheets/space_ship.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
+.space {
+  height: 300px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(bg.gif);
+}
+
+img{
```
Comment (Frcomparan) [inline app/assets/stylesheets/space_ship.css]: Also add a blank space between your tags and the curly braces

**Review by Frcomparan — APPROVED** [review]
LGTM!!

**Review by DiegoSteve — APPROVED** [review]

**Comment by DiegoSteve** [comment]
LGTM ;)

**Pull request merged**